examples/dev/dummy_app/filter_apps.ipynb

```diff
@@ -0,0 +1,72 @@
+{
```

Comment:
This notebook doesn't show anything about filtering. Did you mean to show some other notebook?

Reply:
updated - this PR is just UI side changes, but the notebook is to give you easy apps with metadata to play with.